projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.ts

```diff
@@ -112,6 +112,48 @@ export class IgxSummaryCellComponent {
         return this.grid.resourceStrings[`igx_grid_summary_${summary.key}`] || summary.label;
     }
 
+    /** 
+     * @hidden @internal
+     */
+    public formatSummaryResult(summary: IgxSummaryResult): string {
+        if (summary.summaryResult === undefined || summary.summaryResult === null || summary.summaryResult === '') {
+            return '';
+        }
+
+        if (this.summaryFormatter) {
+            return this.summaryFormatter(summary, this.column.summaries);
+        }
+
+        const args = this.column.pipeArgs;
+        const locale = this.grid.locale;
+
+        if (summary.key === 'count') {
+            return formatNumber(summary.summaryResult, locale)
+        }
+
+        if (summary.defaultFormatting) {
+            switch (this.column.dataType) {
+                case GridColumnDataType.Number:
+                    return formatNumber(summary.summaryResult, locale, args.digitsInfo);
+                case GridColumnDataType.Date:
+                case GridColumnDataType.DateTime:
+                case GridColumnDataType.Time:
+                    return formatDate(summary.summaryResult, args.format, locale, args.timezone);
+                case GridColumnDataType.Currency:
+                    const currencyCode = args.currencyCode
+                        ? args.currencyCode
+                        : getLocaleCurrencyCode(locale);
+                    const currencySumbol = args.display
+                        ? args.display
+                        : getLocaleCurrencySymbol(locale);
+                    return formatCurrency(summary.summaryResult, locale, currencySumbol, currencyCode, args.digitsInfo);
```

Comment:
Can just use`this.currencyCode` instead of calculating it again here. Same goes for urrencySymbol - may be extracted in a getter method.